src/lib/icon/Icon.tsx

```diff
@@ -125,7 +125,7 @@ export const Icon: React.FC<{
   const IconSvg = icons[name];
 
   return (
-    <span className={classNames(`icon icon-${name}`, className)} {...props}>
+    <span className={classNames('icon', `icon-${name}`, className)} {...props}>
```

Comment:
🤷‍♂️

Reply:
I was playing around with adding another classname in there, that's why I did this, it was fine either way.